Frends.Powershell/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="System.Management.Automation.dll" version="10.0.10586.0" targetFramework="net452" />
```

Comment:
This probably is not the official package? Is there an official nuget package?

Reply:
Removing